optimade/validator/data/__init__.py

```diff
@@ -16,21 +12,37 @@
 
 from pathlib import Path
 
-__all__ = ["MANDATORY_FILTER_EXAMPLES"]
+__all__ = ["MANDATORY_FILTER_EXAMPLES", "OPTIONAL_FILTER_EXAMPLES"]
 
 ALIASES = {"values": '"1", "2", "3"', "value": "1", "inverse": "1"}
 
-with open(Path(__file__).parent.joinpath("optional_filters.txt"), "r") as f:
-    OPTIONAL_FILTER_EXAMPLES = [line.strip() for line in f.readlines()]
 
-with open(Path(__file__).parent.joinpath("filters.txt"), "r") as f:
-    _filters = []
-    for line in f.readlines():
-        if line.strip() in OPTIONAL_FILTER_EXAMPLES:
-            continue
-        for alias in ALIASES:
-            if alias in line:
-                line = line.replace(alias, ALIASES[alias])
-        _filters.append(line)
+def _load_filters_and_apply_aliases(path):
+    """ Load a text file containing example filters with one
+    filter per line, and apply aliases to swap out dummy values
+    with more concrete examples.
 
-    MANDATORY_FILTER_EXAMPLES = _filters
+    Parameters:
+        path (str/Path): the filename to load.
+
+    Returns:
+        list: a list of filters.
+
+    """
+    with open(path, "r") as f:
+        _filters = []
+        for line in f.readlines():
+            for alias in ALIASES:
+                if alias in line:
+                    line = line.replace(alias, ALIASES[alias])
```

Comment:
No need to check if it's there I guess? Unless it takes longer for the `replace` method than it does for the the `if`-sentence.
```suggestion
                line = line.replace(alias, ALIASES[alias])
```

Reply:
Imagine it's pretty marginal either way, but don't think its worth changing this